### ISSUE TITLE
Which layers were used for the reconstruction of the style?

### ISSUE THREAD

**Issue by AndreaCirina** [peiwang062/swag#8]
Hi, I'm studying your work for a university project. I'm trying to reconstruct the style of an image as you mentioned in the final part of the paper, but I'm not getting good results. Are the layers for the reconstruction the same as those used for the style transfer? Is the softmax function applied to feature maps or are they multiplied by the constant? Thanks in advance for your reply.

**Comment by peiwang062** [comment]
Yes, the same content layer, 'conv3_5'.
softmax function applied.

**Comment by AndreaCirina** [comment]
Thanks for the reply. Did you use a different combination for the synthesis texture or did you always use the same layers indicated for the style transfer? Softmax function applied to all layer results or not?

**Comment by peiwang062** [comment]
For the texture synthesis, we used all intermediate layers, different from style transfer, following the setting of this paper 'Texture Synthesis Using Convolutional Neural Networks' (should be, although a little bit vague to me now)
Yes, softmax was applied, but just on the conv3_x (x:0->5), conv4_x (x:0->2).

**Comment by AndreaCirina** [comment]
@peiwang062  thanks for your help!
